clang/test/CodeGenSYCL/Inputs/sycl.hpp

```diff
@@ -242,7 +242,7 @@ struct get_kernel_name_t<auto_name, Type> {
   using name = Type;
 };
 
-namespace experimental {
+namespace ONEAPI {
```

Comment:
`intel::experimental::spec_constant` doesn't have an extension defined, and I think we might confuse developers if we start mixing extensions and non-extensions in the same namespace.  As is, `using sycl::ONEAPI` will import experimental functionality unexpectedly.

Is there a reason not to keep this as `ONEAPI::experimental` after the rename?  Otherwise I think somebody should define extensions for these things.

Same comment applies to `intel::experimental::printf`.

Reply:
Ok.  I'd prefer to make it ONEAPI::experimental over intel::experimental.